packages/appstore/src/components/add-app/stories/add-app.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import type { Meta, Story } from '@storybook/react';
+import AddApp from '../index';
+
+type AddAppProps = Parameters<typeof AddApp>[0];
+
+export default {
+    title: 'AddApp',
+    argTypes: {
+        is_dark_mode_on: {
+            description: 'Optional',
+            type: 'boolean',
+            defaultValue: false,
+        },
+        is_mobile: {
+            description: 'Optional',
+            type: 'boolean',
+            defaultValue: false,
+        },
+    },
+} as Meta<AddAppProps>;
+
+const Template: Story<AddAppProps> = args => <AddApp {...args} />;
+
+export const AddAppTemplate = Template.bind({});
+AddAppTemplate.args = {
+    apps: [
+        {
+            app_type_title: 'Deriv X Apps',
+            linked_apps: [
```

Comment:
```suggestion
            account_types: [
```
It's named as `account_types` as per our existing codebase